src/electos/ballotmaker/ballots/contest_data.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass, field
+from typing import List
+
+
+@dataclass
+class BallotMeasureData:
+    """Retrieve ballot measure contest data from a dict"""
+
+    _b_measure_con: dict = field(repr=False)
+    id: str = field(init=False)
+    title: str = field(init=False)
+    district: str = field(init=False)
+    text: str = field(init=False)
+    choices: list = field(default_factory=list, init=False, repr=True)
+
+    def __post_init__(self):
+        self.id = "no_id_provided"
+        self.title = self._b_measure_con["title"]
+        self.district = self._b_measure_con["district"]
+        self.text = self._b_measure_con["text"]
+        self.choices = self._b_measure_con["choices"]
+        # for choice_data in _choices:
+        #     self.choices.append(ChoiceData(choice_data))
+
+
+@dataclass
+class ChoiceData:
+    _choice_data: dict = field(repr=False)
+    id: str = "no_id_provided"
+    label: str = field(init=False)
```

Comment:
You can drop the `__post_init__` if you use:

```
       label: str = field(init=False, default="no label provided")
```

Reply:
I'm going to refactor this code when choice IDs are available, so let's revisit this again.

Reply:
Choice IDs are available as of the merge of #106: [See ballot_lab_data.py:214](https://github.com/TrustTheVote-Project/BallotLab/blob/db0773e3/src/electos/ballotmaker/demo_data/ballot_lab_data.py#L214)